src/history/index.js

```diff
@@ -0,0 +1,42 @@
+import path from 'path';
+import fsApi from 'fs';
+const fs = fsApi.promises;
+
+import * as git from './git.js';
+
+export const ROOT_DIRECTORY = path.resolve() + '/data';
+export const RAW_DIRECTORY = `${ROOT_DIRECTORY}/raw`;
+
+export async function persistRaw(serviceProviderId, policyType, fileContent) {
+  await storeRaw(serviceProviderId, policyType, fileContent);
+  return await commitRaw(serviceProviderId, policyType);
+}
+
+export async function storeRaw(serviceProviderId, policyType, fileContent) {
+  const dir = `${RAW_DIRECTORY}/${serviceProviderId}`;
+
+  if (!fsApi.existsSync(dir)){
+    fsApi.mkdirSync(dir);
+  }
+
+  return fs.writeFile(`${dir}/${policyType}.html`, fileContent);
+}
+
+export async function commitRaw(serviceProviderId, policyType) {
```

Comment:
We're exposing a more abstract function, let's make sure the name reflects the intent rather than the action 🙂 

```suggestion
export async function updateRaw(serviceProviderId, policyType) {
```

Reply:
After discussion: this `export` is only for testing, it is not expected to be exposed. Maybe add a comment? 🙂